Library/Homebrew/test/cleaner_spec.rb

```diff
@@ -20,7 +20,8 @@
 
       subject.clean
 
-      expect((f.bin/"a.out").stat.mode).to eq(0100555)
+      mach_executable_perm = OS.linux? ? 0100444 : 0100555
```

Comment:
Could we set these to the same in the tarball instead?

Reply:
The file permissions in the tarball are the same. This test is related to a file outside of the tarball,`test/support/fixtures/mach/a.out` is a Mach-O file. On Linux, `Cleaner::executable_path?` changes ELF files and shell scripts to executable, but not Mach-O files. Options: I can create a separate `test/support/fixtures/elf` directory for Linux. Alternatively, I can simply disable this particular test on Linux using `:needs_macos`.

Reply:
> I can create a separate test/support/fixtures/elf directory for Linux.

Sounds good 👍

Reply:
Done!